### DISCUSSION TITLE
Merging Split Communication Protocols on EV Converted F150

### DISCUSSION THREAD

**Discussion by BlueForceEnergy** [collin80/SavvyCAN#816] [Q&A]
Hi folks,

The company I am working for has recently converted a 2019 F150 to electric by using a split communication protocol system and I was wondering how it would be possible to combine both systems into one.  The split system we are currently using is working perfectly, but combining them would make things much easier for us once we launch the conversion solution. 

@collin80 would you be able to contact me directly to discuss this and other opportunities further?

All the best.

**Comment by mitchdetailed** [comment]
is this for an electric utility company in Canada ?

**Reply by BlueForceEnergy** [reply]
No this project isn't, but we are gearing this solution towards fleet conversions

**Comment by mitchdetailed** [comment]
rephrase your objective, are you simply trying to merge 2 can buses?

**Reply by BlueForceEnergy** [reply]
Yes

**Comment by mitchdetailed** [comment]
if it's purely a bitrate issue, i sell a 3 channel CAN Gateway that can
bridge 3 canbuses any route. if it's something more complex and you're
comfortable in C, my framework is open sourced . it's at
mintonperformance.com
